### PR TITLE
shellcheck compliance

### PR DESCRIPTION
**Pull request by maghuro**

### CODE REVIEW

**Review by xkill — COMMENTED** [review]
I think that it will be better if you add a disambiguation braces on all the custom vars, for example:
```bash
${wget_optons}
${fwsiteSQ}/${releasenote_file}
${releasenote_path}
[....]
```

**Comment by JackMerlin** [comment]
If it works, I think there is no reason to change it.

File: release/src/router/rom/webs_scripts/merlin_webs_note.sh [hunk 1]
```diff
@@ -31,23 +31,27 @@ else
...
-if [ "$forsq" == "1" ]; then
+if [ "$forsq" = "1" ]; then
 	echo "---- download SQ release note $fwsiteSQ/$releasenote_file ----" >> /tmp/webs_note.log
-	wget $wget_options $fwsiteSQ/$releasenote_file -O $releasenote_path
-	if [ "$?" != "0" ]; then
-		wget $wget_options $fwsiteSQ/$releasenote_file_US -O $releasenote_path
+	if ! wget "$wget_options" $fwsiteSQ/"$releasenote_file" -O $releasenote_path; then
+		wget "$wget_options" $fwsiteSQ/"$releasenote_file_US" -O $releasenote_path
+	else
+	        wget "$wget_options" $fwsiteSQ/"$releasenote_file" -O $releasenote_path
```
Comment (kolyshkin) [inline release/src/router/rom/webs_scripts/merlin_webs_note.sh:39]: This is obviously wrong, as you're downloading the same file twice

File: release/src/router/rom/webs_scripts/merlin_webs_note.sh [hunk 2]
```diff
@@ -31,23 +31,27 @@ else
...
 	echo "---- $fwsiteSQ/$releasenote_file ----" >> /tmp/webs_note.log
 else
 	echo "---- download real release note for $1 ----" >> /tmp/webs_note.log
-	wget $wget_options $fwsite/$releasenote_file -O $releasenote_path
-	if [ "$?" != "0" ]; then
-		wget $wget_options $fwsite/$releasenote_file_US -O $releasenote_path
+	if ! wget "$wget_options" $fwsite/"$releasenote_file" -O $releasenote_path; then
+		wget "$wget_options" $fwsite/"$releasenote_file_US" -O $releasenote_path
+	else
+	        wget "$wget_options" $fwsite/"$releasenote_file" -O $releasenote_path
```
Comment (kolyshkin) [inline release/src/router/rom/webs_scripts/merlin_webs_note.sh:47]: ditto

**Pull request closed without merging**